src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -202,6 +205,46 @@ export class RoslynLanguageServer {
                 workspace: {
                     configuration: (params) => readConfigurations(params),
                 },
+                resolveCodeAction: async (codeAction, token, next) => {
```

Comment:
So I like the approach you took here, but it is slightly different from what we talked about before.  I like this better anyway though, but we can simplify a little bit more.

Since you're directly applying the workspace edit (via `vscode.workspace.applyEdit`) and *not* returning a resolved code action to vscode, it would be better to have this implementation just be a command.

Essentially:
1.  On the server side, instead of adding the flavor to the 'data' object, you would return a fix all code action that contains a command (e.g. `roslyn.fixAll`) with arguments containing whatever flavor information you need to call back to the server as you do now.  An example of where we do this is here - https://github.com/dotnet/roslyn/blob/main/src/Features/LanguageServer/Protocol/Handler/CodeLens/CodeLensHandler.cs#L149
2.  Instead of implementing this as part of the `resolveCodeAction` middleware - you would move this implementation to a command handler.  The implementation here basically wouldn't change (minus my other feedback).  For example, where the client implements the command I linked above - https://github.com/dotnet/vscode-csharp/blob/main/src/lsptoolshost/unitTesting.ts#L21

Then what happens is that when you select the fix all action in the list - the vscode LSP client will invoke our command handler (based on the command id we returned in the code action)